config/engines.yml

```diff
@@ -25,6 +25,12 @@ duplication:
   community: false
   enable_regexps:
   default_ratings_paths:
+  default_config:
```

Comment:
It might be good enough to extract out a `&defaults` reference, so the engine keys can specify less if they care about less.

```yaml
defaults: &defaults
  beta: false
  community: false
  enable_regexps:
  default_ratings_paths:
  default_config:

gofmt:
  <<: *defaults
  image: ...
```